### DISCUSSION TITLE
contribution appearance

### DISCUSSION THREAD

**Discussion by Dkrishnamohanpanda** [community/community#141563] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i am using intelij ide and trying to push my code in to my existing repo . i have succesfully pushed but it is not appear on my contribution list.

-> In my intelij ide i have created separate package and class for my repo but it is not appear[
![Screenshot 2024-10-15 174519](https://github.com/user-attachments/assets/0eaf15e6-30e7-4442-9605-b7a111eb3476)
](url)

**Comment by tmduy** [comment]
Did you add your source code to the stage before commit? I mean "git add 'your-source-code'" (or 
git add ." if you want to add everything).

**Reply by Dkrishnamohanpanda** [reply]
yes

**Comment by mctrinh** [comment]
Which operating system did you use to commit your code?

I encountered a similar issue when committing from a Linux workstation. The workstation's time was set incorrectly, causing the commits to be unrecognized due to the time discrepancy.

The solution is to adjust the time on the Linux machine to the correct local time. I hope this helps!

**Reply by Dkrishnamohanpanda** [reply]
i am using winows did i need to reset my cpu timimg
![Screenshot 2024-10-16 080547](https://github.com/user-attachments/assets/a7f36f73-a4fb-4cd2-9f94-73939a48d5af)

**Comment by mctrinh** [comment]
If you’re using Windows, check your system time and ensure it’s set correctly. You don’t need to reset your CPU timing; just make sure the date and time are accurate in your settings.

Once that’s done, try making a new commit to see if it appears on your contributions list.

If it doesn’t work after that, try clearing the cache and logging in again from the CLI or GitHub Desktop.

You might also reset IntelliJ IDEA, and then make a new commit to see if it shows up.

**Reply by Dkrishnamohanpanda** [reply]
i have checked all of that but still facing the same issue
